scripts/js/settings-api.js

```diff
@@ -212,13 +214,13 @@ function deleteMultipleSessions(ids) {
 
   // Exploit prevention: Return early for non-numeric IDs
   for (const id of ids) {
-    if (Object.hasOwnProperty.call(ids, id) && isNaN(ids[id])) return;
+    // TODO Fix eslint
+    // eslint-disable-next-line unicorn/prefer-number-properties
+    if (Object.hasOwn(ids, id) && isNaN(ids[id])) return;
```

Comment:
What are the exact values we are trying to guard here? `Number.isNan` is a more strict and modern method but we need to make sure it works in our cases. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN#description for the differences to the global `isNan` method.